applications/luci-app-pbr/Makefile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017-2022 Stan Grishin (stangri@melmac.ca)
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_MAINTAINER:=Stan Grishin <stangri@melmac.ca>
+PKG_VERSION:=1.0.0-1
+
+LUCI_TITLE:=Policy Based Routing Service Web UI
+LUCI_DESCRIPTION:=Provides Web UI for Policy Based Routing Service.
+LUCI_DEPENDS:=+luci-base +jsonfilter +pbr
+LUCI_PKGARCH:=all
+
```

Comment:
I am thinking, shouldn't ``luci-app-br`` provide ``luci-app-vpn-routing``? Because pbr provides vpn-policy-routing based on this https://github.com/openwrt/packages/pull/20010/files#diff-6039f5f5edfa818196ebd1777090e34c8a849008989493ada4ed99467b0a3d0fR23 and so it makes sense even for me to have provides for LuCI.